en/ui-system/components/editor/graphics/lineTo.md

```diff
@@ -0,0 +1,24 @@
+# Line To
+
+The `lineTo()` method is used to add a new point, and then create a line from that point to the last specified point in the canvas.
```

Comment:
The `lineTo()` method is used to add a new point, and then create a line from the current graphic cursor to that point.